components/util/write.rs

```diff
@@ -0,0 +1,35 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use rustc_serialize::Encodable;
+use rustc_serialize::json;
+use std::error::Error;
+use std::fs::File;
+use std::io::prelude::*;
+use std::path::Path;
+
+pub fn write_json_to_file<T: Encodable>(data: &T, profile_dir: &str, filename: &str) {
```

Comment:
In which case, it feels more like a storage_trait function rather than a utility function, since it deals with file IO.